api/editor/api-editor/src/main/java/ca/bc/gov/tno/areas/editor/models/ContentModel.java

```diff
@@ -148,6 +149,10 @@ public Content ToContent() {
             .map((file) -> new FileReference(file.getId(), content, file.getPath(), file.getMimeType(), file.getSize(),
                 file.getRunningTime()))
             .toList());
+    content.getContentActions()
```

Comment:
Note that the JSON from the API is simplified to be `content.actions: [{id:  1, value: "true"}]`